Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -228,12 +228,32 @@ extension JNISwift2JavaGenerator {
         printer.println()
       }
 
+      printToStringMethod(&printer, decl)
```

Comment:
Should we also add a toDebugString() which would call the DebugStringConvertible (`String(reflecting: self)`)?